spec/order_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'order'
+=begin
```

Comment:
Style/BlockComments: Do not use block comments. (https://github.com/bbatsov/ruby-style-guide#no-block-comments)